src/views/Task/Update.js

```diff
@@ -1,29 +1,192 @@
-import React from 'react'
+import React, { Component } from 'react'
 import Layout from '../../components/Layout'
+import task from '../../api/task'
+import DatePicker from '../../components/DatePicker'
+import moment from 'moment'
+import Alert from '../../components/Alert'
+import { withRouter } from 'react-router-dom'
+import camelcase from 'camelcase'
 
-export default () => (
-  <Layout>
-    <h2>Edit Task</h2>
-    <label>
-      Title
-      <input type="text" />
-    </label>
-    <label>
-      Start Date
-      <input type="date" />
-    </label>
-    <label>
-      Deadline
-      <input type="date" />
-    </label>
-    <label>
-      Description
-      <textarea />
-    </label>
-    <label>
-      Urgency
-      <input type="text" />
-    </label>
-    <input type="submit" value="Save Changes" />
-  </Layout>
-)
+export default withRouter(class Update extends Component {
+  state = {
+    startDate: moment(),
+    endDate: moment(),
+    title: "",
+    description: "",
+    urgency: "",
+    error: null
+  }
+
+  fields = [
+    //["label", "Type", Validation method]
+    ["Title", "text", v => v.length > 0, false],
```

Comment:
This is failing right now, but should work once @MadsDueKristensen  adds Title to the server-side of Task, it should be good again